lib/player.rb

```diff
@@ -1,10 +1,40 @@
+require_relative './ui.rb'
+
 class Player
-  attr_accessor :name, :sign, :positions
+  include UI
```

Comment:
There are still some lines in this Player class that you can include in the UI module. But I'd leave you to do that on your free time.